modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -313,8 +318,9 @@ private String _buildFragmentHTML(
 	}
 
 	private String _getAPIURL(
-		ObjectEntry fdsEntryObjectEntry,
-		HttpServletRequest httpServletRequest) {
+			ObjectEntry fdsEntryObjectEntry, Set<ObjectEntry> fdsFieldsSet,
```

Comment:
@dsanz @markocikos I need help with this [request](https://github.com/brianchandotcom/liferay-portal/pull/145040#issuecomment-1847892697). Any clue?

Reply:
@juanjofgliferay See how all variable of `ObjectEntry` type are named, `fdsEntryObjectEntry`, `fdsViewObjectEntry` etc. In your case, `fdsFieldsSet` does not match that pattern. It should be `fdsFieldObjectEntries`, with method being `_getFDSFieldObjectEntries`. Sorry for missing this! :persevere:

Reply:
Ok, no worries. Thx